docker/nginx/.htpasswd

```diff
@@ -0,0 +1 @@
+web:$apr1$mK6z5n.S$UWux7wb4NpHWT6YDnqrSx0
```

Comment:
Putting credentials in a public git repo does not seem like a great idea.

Reply:
I agree. Which is why it is not our password!